App-Service-Environment-AzFirewall/azuredeploy.json

```diff
@@ -57,7 +57,7 @@
         },
         "_artifactsLocation": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/App-Service-Environment-AzFirewall/",
+            "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/App-Service-Environment-AzFirewall/",
```

Comment:
revert this change - Azure needs to be capitalized

Reply:
Fixed